src/modules/setup/index.ts

```diff
@@ -1,17 +1,67 @@
-import { getManifest, getAppRoot } from '../../manifest'
+import chalk from 'chalk'
+import log from '../../logger'
+import { getAppRoot, getManifest } from '../../manifest'
+import { IOAppTypesManager } from './IOAppTypesManager'
 import { setupESLint } from './setupESLint'
 import { setupTSConfig } from './setupTSConfig'
-import { IOAppTypesManager } from './IOAppTypesManager'
+import { packageJsonEditor } from './utils'
+
+interface CommandOptions {
+  t?: boolean
+  types?: boolean
+  i?: boolean
+  'ignore-linked'?: boolean
+  o?: boolean
+  'own-types'?: boolean
+}
 
 const buildersToAddAdditionalPackages = ['react', 'node']
 
-export default async (opts: { i?: boolean; 'ignore-linked': boolean }) => {
+const addPostInstallScriptToPackageJson = (folder: string, script: string) => {
+  log.debug(`Add postinstall script to ${chalk.bold(folder)} package.json`)
+  let packageJson: any
+  try {
+    packageJson = packageJsonEditor.read(folder)
+  } catch (err) {
+    if (err.code === 'ENOENT') {
+      packageJson = {}
+    } else {
+      throw err
+    }
+  }
+
+  if (packageJson.scripts && packageJson.scripts.postinstall === script) {
+    return
+  }
+
+  packageJson.scripts = {
+    ...packageJson.scripts,
+    postinstall: script,
```

Comment:
This way we may overwrite the user's `postinstall` script. What do you think about the following?
1. Add our own script if the current one is blank
2. Ignore if the current script already contains ours
3. If there is content and our script is not there, ask the user if we can add it at the beginning of the command